tools/create_testgrid_tickets.py

```diff
@@ -111,11 +115,24 @@ def create_jira_ticket(jclient, existing_tickets, test_id, test_data):
     return new_issue
 
 
+def is_test_failing(test_data):
+    failing = test_data['overall_status'] == "FAILING"
+
+    # testgrid has a bug where it marks successful tests as "FAILING", just because they
+    # have failed test cases. But sometimes test cases are allowed to fail while the run itself is
+    # "passing", but testgrid doesn't take that into account - so it marks a grid as "FAILING" even though
+    # its "Overall" row is green. When the "Overall" row is green but the test is marked as failing, the 
+    # overall tests don't get included in the `tests` list, so we use that to detect this scenario.
+    actually_failing = len(get_overall_tests(test_data)) > 0
```

Comment:
So let's just make this comparison
Having both failing and actual_failing is just super confusing

Reply:
Fixed. I think this is super confusing because the bug is super confusing, my 2 variables are not the problem :upside_down_face:

Reply:
In case of duality due to some outside system, there's no need to add additional duality inside, that's all I'm saying 🙂